src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -462,7 +462,7 @@ async Task IDurableOrchestrationClient.RewindAsync(string instanceId, string rea
         }
 
         /// <inheritdoc />
-        async Task<DurableOrchestrationStatus> IDurableOrchestrationClient.GetStatusAsync(string instanceId, bool showHistory, bool showHistoryOutput, bool showInput)
+        async Task<DurableOrchestrationStatus> IDurableOrchestrationClient.GetStatusAsync(string instanceId, bool showHistory, bool showHistoryOutput, bool showInput, bool showHistoryInput)
```

Comment:
Rather than introducing a new parameter, let's simply rely on the existing `showInput` parameter. Developers probably won't understand the difference between `showInput` and `showHistoryInput` anyways. I think `showInput` can safely apply to both scenarios. This is preferable so that we don't need to introduce any new surface area. The downside is that it changes the behavior of an existing API, but I think it's okay because it enables a behavior that is much more obvious.